admin/class-meta-columns.php

```diff
@@ -70,6 +70,7 @@ public function setup_hooks() {
 		}
 
 		add_filter( 'request', [ $this, 'column_sort_orderby' ] );
+		add_filter( 'default_hidden_columns', [ $this, 'column_hidden' ], 10, 2 );
```

Comment:
If the `$screen` parameter is removed from the `column_hidden` method, this `add_filter` call has to be updated as well to reflect that the callback only accepts one parameter:
```suggestion
		add_filter( 'default_hidden_columns', [ $this, 'column_hidden' ], 10, 1 );
```